api/app/controllers/spree/api/v2/storefront/account_controller.rb

```diff
@@ -43,7 +43,7 @@ def user_create_params
           end
 
           def user_update_params
-            params.require(:user).permit(permitted_user_attributes)
+            params.require(:user).permit(permitted_user_attributes, :saved_locale)
```

Comment:
I think :saved_locale should be added as permitted_user_attributes, but I couldn't find where it is defined.

Reply:
Oh, I've actually found it